src/compiler/generate-common-tests.xsl

```diff
@@ -188,7 +188,9 @@
       </xsl:if>
    </xsl:template>
 
-   <xsl:template match="@href" as="attribute(href)" mode="x:gather-specs">
+   <!-- TODO: Perhaps, @query-at hint should not always be resolved??? (e.g. for MarkLogic) -->
```

Comment:
What is the `@query-at hint`? I don't understand this comment. (I don't use MarkLogic.)

Reply:
I don't understand it either :)
In 319a5b208c8b274d969c57ab46f41dc7b98fa1fe, I added the source of this comment.

Reply:
Thanks, the reference was helpful! I didn't notice that the comment wasn't new. (In the earlier context, the comment a few lines earlier mentions that the query file might be in a database rather than on the file system. Now the TODO comment makes a little more sense to me.)